### PR TITLE
Single pf from config after rebase of main

### PR DESCRIPTION
**Pull request by dkosteck**
Updating code to take the first pf from the config instead of looping over all pfs, newly rebased to main for most recent updates. Copy of the report: https://gist.github.com/dkosteck/2a46499dbecf374647f8493a6d96502e

### CODE REVIEW

File: sriov/tests/SR_IOV_MultipleVFCreation_withMTU/test_SR_IOV_MultipleVFCreation_withMTU.py [hunk 1]
```diff
@@ -5,64 +5,63 @@
 from sriov.common.exec import ShellHandler
 from sriov.common.config import Config
 
-from pytest_html import extras
 from sriov.common.utils import *
 
 LOGGER = logging.getLogger(__name__)
```
Comment (jianzzha) [inline sriov/tests/SR_IOV_MultipleVFCreation_withMTU/test_SR_IOV_MultipleVFCreation_withMTU.py:10]: are we still using the LOGGER?

**Review by jianzzha — COMMENTED** [review]
"import os" can be removed

**Comment by dkosteck** [comment]
Removed redundant imports in https://github.com/redhat-partner-solutions/intel-sriov-test/pull/19/commits/d76119f6de3fea1aa0d1000d61978e8c4715aa85
report: https://gist.github.com/dkosteck/ff2dfbb8b4ff06eeb1bc23597fa2dc2f

**Review by akiselev1 — APPROVED** [review]
Looks good now

**Review by jianzzha — APPROVED** [review]

**Pull request merged**